README.md

```diff
@@ -174,6 +248,28 @@ To change a certificate (root_cert, sub_cert, ARN or pre-existing root cert), yo
 
 This is necessary due to a limitation that prevents certificates from being changed while in use by certain resources.
 
+## Adding external datastore (AWS EFS)
+Users looking to add non-ephemeral storage to their created EC2 instance have the following options; create a new efs as a part of the ec2 deployment stack, or mounting an existing EFS. 
+
+### 1. Create EFS
+Option 1, users have access to the `create_efs` attribute which will create a EFS resource and mount it to the EC2 instance in the application directory at the path: "app_root/data".
+
+> :warning: Be very careful here! The **EFS is fully managed by Terraform**. Therefor **it will be destroyed upon stack destruction**.
+
+### 2. Mount EFS
+Option 2, users have access to the `mount_efs` attributes. Requiring an existing EFS id and optionally a primary security group id the existing EFS will be attached to the ec2 security group to allow traffic.
```

Comment:
```suggestion
Option 2, users have access to the `mount_efs` attributes. Requiring an existing EFS id and optionally a primary security group id the existing EFS will be attached to the ec2 security group to allow traffic.
```

Reply:
The two here are the same but I assume you want the `users` changed to `you`